README.md

```diff
@@ -1,7 +1,56 @@
-# Authgear
+# Authgear Server
+Authgear is an extensible turnkey solution for all of your consumer authentication needs.
+
+We provide a simple auth and self-service portal for your users to start with, customizable as you grow with a simple GraphQL-based Admin API and programmable authentication flows for signup, login, and re-auth.
+
+With a comprehensive range of out-of-the-box features to accelerate your projects, such as:
+
+- Passwordless login: Magic-link/OTP with Email, SMS, WhatsApp; Passkeys
+- Biometric Login on iOS and Android
+- 2FA: TOTP, SMS, Email
+- Integration with analytics, CDP, and drip campaigns
+- Enterprise Security: Audit logs, Bruteforce Protection, Rate Limits
+- B2B Enterprise Connections: ADFS, LDAP
+- and more...
+
+## What is Authgear Server?
+Authgear server is an open-source project that contains the code that powers Authgear's authentication-as-a-service solution. It includes the code for the server, AuthUI, the Portal, and Admin API. You can use Authgear Server to set up your own self-hosted instance of Authgear service.
+
+## Features and Component
+Authgear Server contains the following fears and components of Authgear:
+- Authgear server (the actual service)
+- Portal (A GUI tool for managing Authgear projects)
+- AuthUI
```

Comment:
@buildbro can you add description for AuthUI?

Reply:
OK! I've updated the readme.